etc/docs/generate-docs.ts

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env ts-node
+
+import { parse, stringify } from '@iarna/toml';
+import * as child_process from 'child_process';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import { promisify } from 'util';
+
+const exec = promisify(child_process.exec);
+
+const RELEASES_TOML_FILE = './template/data/releases.toml';
+const RELEASES_JSON_FILE = './template/static/versions.json';
+
+interface JsonVersionSchema {
+  version: string;
+}
+
+interface VersionSchema {
+  version: string;
+  status: 'current' | 'supported' | 'not-supported';
+  api: string;
+  usesMongoDBManual?: boolean;
+  docs?: string;
+  semvarVersion: string;
+}
+
+interface TomlVersionSchema {
+  current: string;
+  mongodDBManual: string;
+  versions: VersionSchema[]
+}
+
+/**
+ * ATTENTION - EDIT THIS BEFORE RELEASING
+ */
+const NEW_VERSION: VersionSchema = {
+  // version - the display name for the version of the driver on the docs website
+  //  ex: Driver 4.5
+  version: 'Driver 4.5',
```

Comment:
Why are we switching this from `<version> Driver` to `Driver <version>`? Won't it look a bit jarring on the index page? Also, if we want to default to generating for the "next" driver version `Next Driver` would probably read better than `Driver Next`, so if we end up taking an argument for this ``` `${semVer} Driver` ``` would probably be better to use as the standard than the other way around

Reply:
This was unintentional.  I'll flip the wording